pkg/installation/installation.go

```diff
@@ -366,15 +366,13 @@ func (i *Installation) applyOverrideFiles() error {
 	for _, file := range i.Options.OverrideConfigs {
 		oFile, err := os.Open(file)
 		if err != nil {
-			fmt.Printf("unable to open file: %s. error: %s\n",
+			return errors.Wrapf(err, "unable to open file: %s. error: %s\n",
```

Comment:
Like this, the error message will appear twice.
this:
`return errors.Wrapf(err, "unable to read data from file: %s. error: %s\n", file, err.Error())`
should be:
`return errors.Wrapf(err, "unable to read data from file: %s.", file)`